hooks/build

```diff
@@ -0,0 +1,4 @@
+docker build --build-arg BASE_IMAGE=openjdk:8-jre -t apache/drill:latest .
+docker build --build-arg BASE_IMAGE=openjdk:11-jre -t apache/drill:openjdk11 .
+docker build --build-arg BASE_IMAGE=openjdk:17 -t apache/drill:openjdk17 .
```

Comment:
We don't support JDK 17 yet: https://github.com/apache/drill/blob/master/.github/workflows/ci.yml#L36